03_launch_mgmt_cluster.sh

```diff
@@ -113,9 +147,36 @@ function launch_cluster_api() {
     popd
 }
 
+#
+# Launch the cluster-api-controller manager (v1alpha2) in the system namespace.
+#
+
+function launch_core_cluster_api() {
+    pushd "${CAPIPATH}"
+      make generate
+      sed -i'' 's/capi-system/metal3/' config/default/kustomization.yaml
```

Comment:
maybe the namespace could be created instead of doing this sed command

Reply:
Creating the namespace is not an issue since that is taken care of by the corresponding installation processes for each repo. I kept this under 'metal3' since other controllers and components were defined in the same namespace. Besides a different namespace for each controller would require changing the verify script also since then verification of deployments , statefulsets etc would require the namespace parameter.